### ISSUE TITLE
facenet中triplet_loss实现

### ISSUE THREAD

**Issue by WangTaoSpace** [PaddlePaddle/Paddle#12503]
pos_dist = tf.reduce_sum(tf.square(tf.subtract(anchor, positive)), 1)
        neg_dist = tf.reduce_sum(tf.square(tf.subtract(anchor, negative)), 1)
        
        basic_loss = tf.add(tf.subtract(pos_dist,neg_dist), alpha)
        loss = tf.reduce_mean(tf.maximum(basic_loss, 0.0), 0)

我想在paddle中实现上面的loss，但是
paddle 0.11.0 v2.layer 我没有找到向量减法和tf.maximum 这个方法的替代。

**Comment by qingqing01** [comment]
@WangTaoSpace 建议使用Fluid，trilet_loss使用Fluid的op可以很容易拼出来，我们自己也有实现，后续也会release出来。

你上面列的 reduce_sum/mean, square, subtriact, maximum在Fluid都有。  Fluid支持 + - * / 表达式。

**Comment by lucywsq** [comment]
您好，此issue在近一个月内暂无更新，我们将于今天内关闭。若在关闭后您仍需跟进提问，可重新开启此问题， 我们将在24小时内回复您。因关闭带来的不便我们深表歉意，请您谅解~感谢您对PaddlePaddle的支持!
